cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -25,6 +25,8 @@ import (
 )
 
 const (
+	// BuildIDOSEnv is the os env name to get build id
+	BuildIDOSEnv           = "BUILD_ID"
 	translateFailedPrefix  = "TranslateFailed"
 	privacyInfoReplacement = "[Privacy Info]"
```

Comment:
Why do we need this replacement? It doesn't provide any useful info to users and can be confusing.

Reply:
There can be a hole in the error message sentence. Having a replacement make it easier to read.

Reply:
Do you have some examples of holes in error messages?

Also, if Privacy section is at the end of the message, there won't be any hole and we don't need the placeholder.

Reply:
I did modified strings to the format as you said, just wanted to have a double insurance. If you think it's useless, I am OK to remove it.